content/kubermatic/main/tutorials-howtos/cluster-rbac/_index.en.md

```diff
@@ -0,0 +1,108 @@
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster-admin       | Allows admin access. Allows read/write access to most resources in a namespace, including creating roles and role bindings within the namespace. This role does not allow write access to resource quota or to the namespace itself. |
+| edit                | Allows read/write access to most objects in a namespace. This role does not allow viewing or modifying roles or role bindings. However, this role allows accessing secrets and running pods as any service account in the namespace  |
+| view                | Allows read-only access to see most objects in a namespace. It does not allow viewing roles or role bindings.                                                                                                                        |
+| list-namespaces     | Allows to list namespaces                                                                                                                                                                                                            |
+
+**Permission on namespace level:**
+
+| Default Role     | Description                                                                                                                                         |
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| namespace-admin  | Allows admin access. Allows read/write access to most resources in a namespace.                                                                     |
+| namespace-editor | Allows read/write access to most objects in a namespace. This role allows accessing secrets and running pods as any service account in the namespace|
+| namespace-viewer | Allows read-only access to see most objects in a namespace.                                                                                         |
+
+
+
+# Manage User's permissions
+You can grant permission to a group by clicking on `add Bindings`.
+![Grant permission to a user](/img/kubermatic/main/ui/rbac_user_binding.png?classes=shadow,border "Grant permission to a user")
+
+{{% notice info %}}
+The cluster owner is automatically connected to the `admin` cluster role.
+{{% /notice %}}
+
+# Manage Group's permissions
+Group are named collection of users. You can grant permission to a group by clicking on `add Bindings`.
+![Grant permission to a group](/img/kubermatic/main/ui/rbac_group_binding.png?classes=shadow,border "Grant permission to a Group")
+
+In this example, we grant the permission `view` on the cluster to the `oidc` group `security-audit`
```

Comment:
```suggestion
In this example, we grant the role `view` on the cluster to the OIDC group `security-audit`
```

Reply:
OIDC isn't used as a code reference here or something like that, so let's use the "official" spelling.